tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "noFallthroughCasesInSwitch": true,
     "noUnusedParameters": false,
     "noUnusedLocals": false,
+    "skipLibCheck": true,
```

Comment:
Why do we need this?

Reply:
Because graphql-ws previously used the `WebSocket` and `CloseEvent` types from the global scope which does not exist outside the "dom" lib. However, this is not the case anymore as of [v1.9.3](https://github.com/enisdenjo/graphql-ws/releases/tag/v1.9.3), so I removed it and the config is now unchanged.

Reply:
Awesome; thanks :raised_hands: